ocs_ci/ocs/ocp.py

```diff
@@ -932,6 +932,21 @@ def get_ocs_version():
     return ocp_cluster.get()['items'][0]['spec']['version']
 
 
+def get_ocs_parsed_version():
+    """
+    Returns ocs version as float
+    Returns:
+        float: ocs version number as major.minor (for example: 4.5)
+
+    """
+    ocs_ver = get_ocs_version().split("-")
+    major_minor = ocs_ver[0].split(".")
+    major = major_minor[0]
+    minor = major_minor[1]
+
+    return float(f"{major}.{minor}")
```

Comment:
Will `major_minor` ever include a patch as well? For example, "4.5.1" or will it always be like "4.5"? If it does not include a patch then we don't need to split this and re-join it in the fstring. We can simply return the float.

```suggestion
    major_minor = ocs_ver[0].split(".")
    return float(major_minor)
```